cobra/core/model.py

```diff
@@ -500,8 +500,22 @@ def add_boundary(self, metabolite, type="exchange", reaction_id=None,
         >>> demand.build_reaction_string()
         'atp_c --> '
         """
-        types = dict(exchange=("EX", -ub, ub), demand=("DM", 0, ub),
-                     sink=("SK", -ub, ub))
+        ub = CONFIGURATION.upper_bound if ub is None else ub
+        lb = CONFIGURATION.lower_bound if lb is None else lb
+        types = {
+            "exchange": ("EX", -ub, ub),
```

Comment:
It would be a behavioral change but this should probably become for consistency with the configuration.

```suggestion
            "exchange": ("EX", lb, ub),
```

Reply:
Yeah, actually that is missing some flexibility since you can not specify a sole import or export reaction (lb=0 or ub=0). But the behavior you suggest makes sense!